### DISCUSSION TITLE
lokale Ports öffnen

### DISCUSSION THREAD

**Discussion by edeso** [Freetz-NG/freetz-ng#160] [Allgemein] [answered]
hey,

https://freetz-ng.github.io/freetz-ng/make/dropbear.html erwähnt 
```
1. Paket CGI/AVM-Firewall mitinstallieren
```
was allerdings mit akt. FritzOS >5.5 im menuconfig deaktiviert ist. gibt es einen anderen Weg? ist der irgendwo dokumentiert?

danke!

**Comment by harryboo** [comment]
AVM-portfw installieren

**Comment by fda77** [comment, marked as answer]
Je nach Fritzos Version wird ein anderes Paket davon angezeigt

 - AVM-firewall
 - AVM-forwarding
 - AVM-portfw

Auf jeder Seite gibt es unten ein "Improve this page.", falls das jemand erweitern mag
